cpp/models/abm/world.h

```diff
@@ -447,6 +470,75 @@ class World
     }
     /** @} */
 
+    /**
+     * @brief Flip activeness status of a person in the world.
+     * @param[in] person_id PersonId of Person whose activeness status is fipped.
+     */
+    void change_activeness(PersonId person_id)
+    {
+        m_activeness_statuses[person_id.get()] = !m_activeness_statuses[person_id.get()];
+    }
+
+    /**
+     * @brief Copy the persons from another World to this World. 
+     * If the persons are at a location in this world they are activated, otherwise they are deactivated.
+     * If necessary the person ids are changed such that they correspond to the index in this world's m_persons vector.
+     * @param[in] other The World the Person%s are copied from.
+     */
+    void copy_persons_from_other_world(const World& other)
+    {
+        for (auto& p : other.get_persons()) {
+            if (p.get_id() != static_cast<uint32_t>(m_persons.size())) {
+                mio::log_debug("In world.copy_persons_from_other_world: PersonId does not correspond to index in "
+                               "m_persons vector. Person is copied with adapted Id");
+            }
+            PersonId new_id{static_cast<uint32_t>(m_persons.size())};
+            m_persons.emplace_back(p, new_id);
+            if (p.get_location_world_id() == m_id) {
+                m_activeness_statuses.push_back(true);
+            }
+            else {
+                m_activeness_statuses.push_back(false);
+            }
+        }
+    }
+
+    /**
+    * @brief Set the Person%s of the World.
+    * @param[in] persons The Person%s of the World.
+    */
+    void set_persons(std::vector<Person>& persons)
```

Comment:
Since this too behaves very similar to `for (p : persons) add_person(p)` I would prefer making this an external function.
Maybe we should consider adding a builder class for World or even the entire World-Graph.